pkg/app/piped/executor/ecs/ecs.go

```diff
@@ -157,6 +157,43 @@ func applyServiceDefinition(ctx context.Context, cli provider.Client, serviceDef
 	return service, nil
 }
 
+func applyStandaloneTask(
+	ctx context.Context,
+	cli provider.Client,
+	taskDefinition types.TaskDefinition,
+	ecsInput *config.ECSDeploymentInput,
+) error {
+	return cli.RunTask(
+		ctx,
+		taskDefinition,
+		ecsInput.ClusterArn,
+		ecsInput.LaunchType,
+		&ecsInput.AwsVpcConfiguration,
+	)
+}
+
+func runStandaloneTask(
+	ctx context.Context,
+	in *executor.Input,
+	cloudProviderName string,
+	cloudProviderCfg *config.PlatformProviderECSConfig,
+	taskDefinition types.TaskDefinition,
+	ecsInput *config.ECSDeploymentInput,
+) bool {
+	client, err := provider.DefaultRegistry().Client(cloudProviderName, cloudProviderCfg, in.Logger)
+	if err != nil {
+		in.LogPersister.Errorf("Unable to create ECS client for the provider %s: %v", cloudProviderName, err)
+		return false
+	}
+
+	err = applyStandaloneTask(ctx, client, taskDefinition, ecsInput)
```

Comment:
In this case, directly call to cli.RunTask and remove applyStandaloneTask could be a better choice 🤔

Reply:
I think you are right.
I was creating unnecessary functions...🙇